pkg/status/status_test.go

```diff
@@ -295,7 +295,7 @@ status:
 			expectedErr:         nil,
 		},
 		{
-			name: "minimal embedded",
+			name: "default",
```

Comment:
can you rename this to something more descriptive?

Reply:
Thanks, renamed.